viewto-frontend/src/models/Scenario.ts

```diff
@@ -58,13 +65,112 @@ export class Scenario extends Model({
     }
 
     onInit() {
+
+        if(this.project){
+            this._loadData();
+        }
         reaction(
-            () => [this.project],
-            () => {
-                if (this.project.id && this.project.model && this.project.version && this.data === undefined) {
-                    this.setLoad(true);
-                }
-            }
-        );
+            () => [this.project.id],
+            () =>{
+                this._loadData();
+            } 
+            );
+
+    }
+
+
+    // this needs to be fetched in the authentaticaiton process
+    private static token = "089149f2176eb6b5ba309f011ca24bcfc32376998e";
+    // this should come from the login process
+    private static url = "https://speckle.xyz";
+    // comes from the commit selection 
+    private static objectRef = "f4b16ebe7e93ea3ec653cd284d72ca05";
+
+    private _loadData() {
+        console.log(`Loading new Project: ${this.project.id}`);
+    
+        const loader = new ObjectLoader({
+            token: Scenario.token,
+            serverUrl: Scenario.url,
+            streamId: this.project.id,
+            objectId: Scenario.objectRef,
+            // options: {
+            //     fullyTraverseArrays: false, // Default: false. By default, if an array starts with a primitive type, it will not be traversed. Set it to true if you want to capture scenarios in which lists can have intersped objects and primitives, e.g. [ 1, 2, "a", { important object } ]
+            //     excludeProps: ["displayValue", "displayMesh", "__closure"], // Default: []. Any prop names that you pass in here will be ignored from object construction traversal.
+            // },
+        });
+    
+        const getObjectAndManuallyConvert = async (): Promise<SpeckleObject | SpeckleObject[]> => {
+        
+            const referenceObj = await loader.getAndConstructObject((e) => {
+                // event loop for getting progress on the loading
+                console.log("Progress ", e.stage, ":", e.current / e.total);
+        });
+
+        
+        // deconstructing the speckle object
+        // @ts-ignore
+        const data = referenceObj.Data;
+
+        // the conversions from speckle to keystone
+        const study = new ViewStudy({

```

Comment:
@ericyoungberg I know we ended our worksession with planning to not create a study model, but I wanted to do it still for practice and potential sparking any new clarity to how the objects work. I added a few more keystone models as well for other view objects too. Seems like this might be overkill.